src/v/cluster/rm_stm.cc

```diff
@@ -34,11 +34,6 @@ static ss::sstring abort_idx_name(model::offset first, model::offset last) {
     return fmt::format("abort.idx.{}.{}", first, last);
 }
 
-static const model::violation_recovery_policy crash{
```

Comment:
> cluster: Delete usless recovery_policy

do you mean "unused" ? why is becoming unused? is it that we now have a semantics that is always stronger than this policy? please explain why the change is being made in the commit message.

Reply:
update the commit mesage